tests/unit/__init__.py

```diff
@@ -0,0 +1 @@
+"""Unit test module."""
```

Comment:
I think that this whole file is not needed.

Reply:
File removed.